examples/using-all-channels/app.php

```diff
@@ -49,7 +50,10 @@
     new Email\Channel(
         new Mailer(new GmailSmtpTransport(getenv('MAILER_USERNAME'), getenv('MAILER_PASSWORD'))),
         Email\MessageGenerator::withTimeToDiff(new DateTimeImmutable()),
-        Email\ByPassEmail::sendAllTo(getenv('MAILER_USERNAME'))
+        new Email\AddressGenerator((new ByPassEmail())
+            ->setSendEmailsToAssignee(false) // <- OverriddenEmails wont have no effect as long as this is false
+            ->setOverriddenEmails(json_decode($_ENV['OVERRIDDEN_EMAILS'], true))
+            ->setSendCopyTo(getenv('MAILER_USERNAME')))
```

Comment:
Is there any difference between `getenv()` and `$_ENV[]`?

Reply:
In this context, actually not, because of these lines
```php 
$dotEnv = Dotenv\Dotenv::create(__DIR__);
$dotEnv->load();
```
what they do is actually load into the `$_ENV` global var the `.env` "key:value" values.
BUT, for consistency, GOOD catch. We should stick to one. In our case the function `getenv()`